database-service/tests/fixtures-generator/entrypoint.sh

```diff
@@ -1,14 +1,23 @@
 #! /bin/bash
 
 FIXTURES_SET=${1:-small}
-FIXTURES_FOLDER=${2:-database-service/fixtures}
+FIXTURES_FOLDER=${2:-./fixtures}
 FIXTURES_MIGRATIONS_FOLDER=${3:-${FIXTURES_FOLDER}/migrations}
+APP_ROOT=${4:-./}
 
-GENERATE_JSON_FIXTURES=database-service/tests/fixtures-generator/generate_json_fixtures.py
-JSON_TO_SQL=database-service/tests/fixtures-generator/json2sql.py
+GENERATE_JSON_FIXTURES=${APP_ROOT}/tests/fixtures-generator/generate_json_fixtures.py
+JSON_TO_SQL=${APP_ROOT}/tests/fixtures-generator/json2sql.py
+
+if [ ! -d ${FIXTURES_MIGRATIONS_FOLDER} ]; then mkdir -p ${FIXTURES_MIGRATIONS_FOLDER}; fi
 
 python ${GENERATE_JSON_FIXTURES} -o ${FIXTURES_FOLDER}
+
+index=$(($(ls ${APP_ROOT}/migrations | tail -n 1 | cut -d "_" -f 1)+1))
```

Comment:
I've had to set the timestamps to a fixed value here (which bases on the timestamps of the migrations located under `database-service/migrations`). That's because the `test_migrations.py` runs all the tests without cleaning up after themselves. Without those fixed timestamps, the same fixtures migrations are applied several times with different timestamps (because they are not applied at the very same time) resulting in errors because we cannot insert the same data twice into the database.